cst438-project-airline/src/main/java/cst438/service/AirlineService.java

```diff
@@ -0,0 +1,40 @@
+package cst438.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import cst438.domain.FlightRepository;
+import cst438.domain.PassengerRepository;
+import cst438.domain.ReservationRepository;
+import cst438.domain.SeatRepository;
+import cst438.domain.UserRepository;
+
+@Service
+public class AirlineService {
+
+  @Autowired
+  FlightRepository flightRepository;
+  @Autowired
+  PassengerRepository passengerRepository;
+  @Autowired
+  ReservationRepository reservationRepository;
+  @Autowired
+  SeatRepository seatRepository;
+  @Autowired
+  UserRepository userRepository;
+
+  public AirlineService() {}
+
+  public AirlineService(FlightRepository flightRepository, PassengerRepository passengerRepository,
+      ReservationRepository reservationRepository, SeatRepository seatRepository,
+      UserRepository userRepository) {
+    super();
+    this.flightRepository = flightRepository;
+    this.passengerRepository = passengerRepository;
+    this.reservationRepository = reservationRepository;
+    this.seatRepository = seatRepository;
+    this.userRepository = userRepository;
+  }
+
```

Comment:
We need the following methods:
- A method to return available flights and seats for a given origin city and destination
   - if all seats are used up, then return "no flights"
   - if there are no flights then we return "no flights"
- A method to book a reservation
- A method to change a reservation
- A method to cancel a reservation